templates/katello.yaml.erb

```diff
@@ -23,19 +23,10 @@
     :ssl_key_file: <%= @candlepin_events_ssl_key %>
     :ssl_ca_file: <%= @candlepin_ca_cert %>
 
+  :katello_applicability: true
```

Comment:
@jlsherrill does it make sense for Katello to become smarter about this? Looks like today it must be set but perhaps this should default to true?

Reply:
I believe eventually there will be more clean up in this file as Pulp 2 is completely removed.